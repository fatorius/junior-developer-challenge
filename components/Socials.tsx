import VALUES from "@/VALUES";
import { Ionicons } from "@expo/vector-icons";
import { TouchableOpacity, View } from "react-native";

function Socials() {
	return (
		<View
			style={{
				marginTop: VALUES.Espacamento,
				flexDirection: "row",
				justifyContent: "center",
			}}>
			<TouchableOpacity
				style={{
					padding: VALUES.Espacamento,
					backgroundColor: VALUES.Tema.cinza,
					borderRadius: VALUES.Espacamento / 2,
					marginHorizontal: VALUES.Espacamento,
				}}>
				<Ionicons
					name="logo-google"
					color={VALUES.Tema.corDoTexto}
					size={VALUES.Espacamento * 2}
				/>
			</TouchableOpacity>
			<TouchableOpacity
				style={{
					padding: VALUES.Espacamento,
					backgroundColor: VALUES.Tema.cinza,
					borderRadius: VALUES.Espacamento / 2,
					marginHorizontal: VALUES.Espacamento,
				}}>
				<Ionicons
					name="logo-apple"
					color={VALUES.Tema.corDoTexto}
					size={VALUES.Espacamento * 2}
				/>
			</TouchableOpacity>
			<TouchableOpacity
				style={{
					padding: VALUES.Espacamento,
					backgroundColor: VALUES.Tema.cinza,
					borderRadius: VALUES.Espacamento / 2,
					marginHorizontal: VALUES.Espacamento,
				}}>
				<Ionicons
					name="logo-facebook"
					color={VALUES.Tema.corDoTexto}
					size={VALUES.Espacamento * 2}
				/>
			</TouchableOpacity>
		</View>
	);
}

export default Socials;
