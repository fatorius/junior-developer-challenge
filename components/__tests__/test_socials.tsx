import React from "react";
import renderer from "react-test-renderer";
import Socials from "../Socials";

test("Render: Socials", () => {
	const tree = renderer.create(<Socials />).toJSON();
	expect(tree).toMatchSnapshot();
});
