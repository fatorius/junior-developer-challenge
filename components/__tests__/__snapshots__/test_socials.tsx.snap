// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render: Socials 1`] = `
<View
  style={
    {
      "flexDirection": "row",
      "justifyContent": "center",
      "marginTop": 9,
    }
  }
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "backgroundColor": "#ECECEC",
        "borderRadius": 4.5,
        "marginHorizontal": 9,
        "opacity": 1,
        "padding": 9,
      }
    }
  >
    <Text />
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "backgroundColor": "#ECECEC",
        "borderRadius": 4.5,
        "marginHorizontal": 9,
        "opacity": 1,
        "padding": 9,
      }
    }
  >
    <Text />
  </View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "backgroundColor": "#ECECEC",
        "borderRadius": 4.5,
        "marginHorizontal": 9,
        "opacity": 1,
        "padding": 9,
      }
    }
  >
    <Text />
  </View>
</View>
`;
